scripts/zones/Port_Jeuno/Zone.lua

```diff
@@ -26,6 +26,13 @@ function onZoneIn(player,prevZone)
         -- No need for an 'else' to change it back outside these dates as a re-zone will handle that.
     end
 
+    if
+        ENABLE_ABYSSEA == 1 and player:getMainLvl() >= 30
```

Comment:
this shouldn't be its own if, its should be in the main collection of if/else that determine cut scene events - its not like you can get 2 of them on the same zone-in.

Reply:
never mind I see the rest of the below block is full a rats nest - will restructure clean up another day